linera-views/src/backends/rocks_db.rs

```diff
@@ -300,21 +303,38 @@ impl RocksDbStoreInternal {
         if !std::path::Path::exists(&path_buf) {
             std::fs::create_dir(path_buf.clone())?;
         }
+        let sys = System::new_with_specifics(
+            RefreshKind::nothing()
+                .with_cpu(CpuRefreshKind::everything())
+                .with_memory(MemoryRefreshKind::nothing().with_ram()),
+        );
+        let num_cpus = sys.cpus().len() as i32;
+        let total_ram = sys.total_memory() as usize;
         let mut options = rocksdb::Options::default();
         options.create_if_missing(true);
         options.create_missing_column_families(true);
         // Flush in-memory buffer to disk more often
-        options.set_write_buffer_size(DB_CACHE_SIZE);
-        options.set_max_write_buffer_number(DB_MAX_WRITE_BUFFER_NUMBER);
+        options.set_write_buffer_size(WRITE_BUFFER_SIZE);
+        options.set_max_write_buffer_number(MAX_WRITE_BUFFER_NUMBER);
         options.set_compression_type(rocksdb::DBCompressionType::Lz4);
-        options.set_level_zero_slowdown_writes_trigger(-1);
-        options.set_level_zero_stop_writes_trigger(48);
-        options.set_stats_dump_period_sec(60);
-        options.enable_statistics();
-        options.increase_parallelism(num_cpus::get() as i32);
-        options.set_max_background_jobs(8);
+        options.set_level_zero_slowdown_writes_trigger(12);
+        options.set_level_zero_stop_writes_trigger(20);
+        options.increase_parallelism((num_cpus / 2).max(1));
+        options.set_max_background_jobs((num_cpus / 3).max(1));
```

Comment:
I think it would be useful to comment on the reason for choosing such parameters.

About commenting vs not commenting a good heuristic is to comment on things that
would take 10 minutes for a developer to find out. Here, I do not see why you chose
num_cpus / 2 and num_cpus / 3. That feels arbitrary and so deserve comment.
take you 10 minutes
If something takes 10 minutes for the developer to find ou